test/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/IERS/Manager.test.cpp

```diff
@@ -55,6 +91,14 @@ class OpenSpaceToolkit_Physics_Coordinate_Frame_Providers_IERS_Manager : public
     Finals2000A finals2000A_ = Finals2000A::Undefined();
 
     Manager& manager_ = Manager::Get();
+
+    const char* localRepositoryVarName = "OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_LOCAL_REPOSITORY";
+    const char* fullDataVarName = "OSTK_PHYSICS_DATA_LOCAL_REPOSITORY";
+    const char* modeVarName = "OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_MODE";
+
+    char* localRepositoryPath;
+    char* fullDataPath;
+    char* modeValue;
```

Comment:
```suggestion
    const char* localRepositoryVarName_ = "OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_LOCAL_REPOSITORY";
    const char* fullDataVarName_ = "OSTK_PHYSICS_DATA_LOCAL_REPOSITORY";
    const char* modeVarName_ = "OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_MODE";

    char* localRepositoryPath);
    char* fullDataPath_;
    char* modeValue_;
```
Nit: should we keep these as underscores as private/protected?